src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudConfig.scala

```diff
@@ -140,8 +140,8 @@ object FireCloudConfig {
     val whitelists: Set[NihWhitelist] = {
       val whitelistConfigs = nih.getConfig("whitelists")
 
-      whitelistConfigs.root.asScala.map { case (name, configObject: ConfigObject) =>
-        val config = configObject.toConfig
+      whitelistConfigs.root.asScala.map { case (name, configObject) =>
+        val config = configObject.asInstanceOf[ConfigObject].toConfig
```

Comment:
I hate `asInstanceOf` ... could you do:
```
      whitelistConfigs.root.asScala.collect { case (name, configObject:ConfigObject) =>
        val config = configObject.toConfig
        ...
```
( I wrote this in a vacuum without testing it )

Reply:
much nicer, thanks!